naturalspeech2_pytorch/aligner.py

```diff
@@ -0,0 +1,145 @@
+from typing import Tuple
+import torch
+from torch import nn
+import torch.nn.functional as F
+from einops import rearrange
+import numpy as np
+from numba import jit, prange
```

Comment:
hmm, was this here for the `maximum_path`?

Reply:
yes i'll  remove the import. we can add if needed